modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -5,50 +5,45 @@ endif::[]
 
 Anypoint Studio allows you to customize your preferences for working with Maven.
 
-image::maven-preferences-reference-7fdf0.png[]
+image::maven-preferences-reference.png["*Maven Preferences Reference* window accessed from the *Settings...* option, from the *Anypoint Studio* menu."]
+
 
 [%header,%autowidth.spread]
 |===
-| Preference |Description |Default Value
+| Preference | Description | Default Value
 | Display runtime version not available warnings
 | When enabled, it warns you when the runtime version configured in your pom.xml file is different than the one configured in your project.
 | Enabled
 
-| Use embedded Maven Installation (Maven 3.3.9)
+| Use embedded Maven Installation
 | When enabled, it uses the Maven installation embedded in Anypoint Studio.
 | Enabled
 
 | Maven Installation home directory
 | The directory of your local Maven installation. +
 You can only edit this preference when you disable Use embedded Maven Installation.
-| /usr/local/Cellar/maven/3.3.9/libexec
-
+| Empty
 
 | Override user settings path
```

Comment:
This option is not on Maven Preferences Page, it is on deeper level in the tree:
<img width="755" alt="Screenshot 2023-01-18 at 16 21 22" src="https://user-images.githubusercontent.com/13531755/213274835-352f2f09-5f07-4bee-b5cf-c01a8d2de33d.png">

Reply:
@mauriarroqui, should we remove the "Override user settings path" from this table?
Is there any other change needed?
Thanks!